blog/2025-11-20-labs-preview-features.md

```diff
@@ -0,0 +1,158 @@
+---
+author: Frenck
+authorURL: https://github.com/frenck
+authorImageURL: https://avatars.githubusercontent.com/u/195327?v=4
+title: "Introducing Labs: Preview features before they become standard"
+---
+
+We're excited to announce a new system for shipping preview features in Home Assistant: **Labs**. Labs provides a standardized way for integrations to offer fully tested features that users can opt into before they become standard, allowing us to gather feedback and refine the design based on real-world usage.
+
+## What are Labs preview features?
+
+Labs preview features are different from beta testing. While beta testing evaluates the stability of upcoming Home Assistant releases, Labs is about refining user interfaces and design. Labs features are fully tested and functional, but their design and behavior may still change as we gather real-world usage and feedback. This means they might have breaking changes, be extended with new functionality, or even be removed if they don't work out.
+
+Think of it this way:
+
+- **Beta**: Evaluates the stability of upcoming Home Assistant releases
+- **Labs**: Fully tested features with evolving design and user interface, refined through real-world usage and feedback
+
+## How it works
+
+Labs features are defined in an integration's `manifest.json`:
```

Comment:
The manifest example is also included below in the how it's implemented section.